dist.sh

```diff
@@ -18,10 +18,7 @@ set -e
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 rm -rf   $DIR/dist/docker
 mkdir -p $DIR/dist/docker
-rm -rf   $DIR/.godeps
-mkdir -p $DIR/.godeps
-export GOPATH=$DIR/.godeps:$GOPATH
-GOPATH=$DIR/.godeps gpm install
```

Comment:
I've always liked that `dist.sh` preserves the state of my `GOPATH`, should we maintain that property?

Reply:
@mreiferson With `dep` you don't have to change `GOPATH`, it uses `./vendor`. Maybe I misunderstood.

Reply:
Perfect!